### ISSUE TITLE
Feature Request: Change tracking policies to now support all regions that are available

### ISSUE THREAD

**Issue by steph409** [Azure/Enterprise-Scale#1703]
**Describe the solution you'd like**

Hi!
The old log analytics agent will be turned off in august 2024 and moving to AMA quickly is critical. Unfortunately, the change tracking built-in policies that you are using in the enterprise scale have a hard-coded list of supported regions. 
Policy definition id: /providers/Microsoft.Authorization/policyDefinitions/4bb303db-d051-4099-95d2-e3e1428a4d2c
<img width="686" alt="image" src="https://github.com/Azure/Enterprise-Scale/assets/33083326/3e4c7d67-f23d-4623-a8e0-0f83f318ec13">

This list seems to be outdated. When looking in the portal at the change tracking, there are other regions which can do change tracking, e.g. Germany west central:
<img width="908" alt="image" src="https://github.com/Azure/Enterprise-Scale/assets/33083326/336405fd-ea46-4edd-82da-f4017c8c3aa2">

Please update the policy so that the change tracking can be rolled out using the enterprise scale, because this is 
blocking migration to the new monitoring agent.

**Comment by arjenhuitema** [comment]
Hi @steph409, thank you for your feedback. 

MMA retires on the 31 of August, however, this does not mean that the agent will _turn of_ please see [Important Note](https://learn.microsoft.com/en-us/azure/azure-monitor/agents/azure-monitor-agent-migration) on what you can expect when you use the MMA or OMS agent after this date. For example, data upload will continue to work for existing agent until February 1 2025.

> - **Data upload:** Cloud ingestion services will gradually reduce support for MMA agents, which may result in decreased support and potential compatibility issues for MMA agents over time. Ingestion for MMA will be unchanged until February 1 2025.

Good point on the Change Tracking policies, those indeed need to be updated. I will reach out internally and keep you updated.

**Comment by Springstone** [comment]
@arjenhuitema @jtracey93 should we not work on removing the `allowedValues` from this built-in to minimize maintenance overhead?

**Comment by arjenhuitema** [comment]
@Springstone yes correct. I got confirmation that `listOfApplicableLocations` is no longer required. This will be implemented over the upcomming weeks:

1. Remove the `allowedValues`
2. Set the `defaultValue `as an empty array.
3. Change the Policy Rule logic to proceed if `listOfApplicableLocations` is empty.

**Comment by Springstone** [comment]
@steph409 FYI, PG is updating the built-in policies. We'll keep this open to track.

**Comment by Springstone** [comment]
Just an update, the update to the built-in policy is under review and is on the way.
